modules/openapi-generator/src/main/resources/python/rest.mustache

```diff
@@ -133,7 +133,7 @@ class RESTClientObject(object):
 
         timeout = None
         if _request_timeout:
-            if isinstance(_request_timeout, (int, ) if six.PY3 else (int, long)):  # noqa: E501,F821
```

Comment:
Thank you for the PR! Can you add this in python-experimental also?

Reply:
Good catch! I missed that we had a `python-experimental` package - is this package intended to solely generate Python3.x compatible code?

Reply:
Python-experimental has better feature coverage
it includes:
- additionalProperties support
- data types are defined with classes not strings
- type checking is done when sending to endpoints, receiving from them, and instantiating models
- composed schemas are fully supported
- python 2 has been removed, it uses python 3.5 or higher

Reply:
Gotcha - thanks for clarifying that. I noticed it appeared to only use Python3.x supported syntax, so I went ahead and followed suit. Glad to hear that was the right move!